arviz/data/inference_data.pyi

```diff
@@ -16,6 +16,9 @@ class InferenceData:
     posterior_predictive: Optional[xr.Dataset]
     predictions: Optional[xr.Dataset]
     predictions_constant_data: Optional[xr.Dataset]
+    _warmup_posterior: Optional[xr.Dataset]
+    _warmup_posterior_predictive: Optional[xr.Dataset]
+    _warmup_predictions: Optional[xr.Dataset]
```

Comment:
Is there a particular reason why not all `SUPPORTED_GROUPS_WARMUP` are here? 

Note: `log_likelihood` and sample stats ones are missing and `observations` should be `observed_data`

Reply:
True, we should add these